Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -203,9 +203,10 @@ - (void) setEnableAutoPurge: (BOOL)enableAutoPurge {
 }
 
 - (CBLDatabase*) database {
-    if (!_database)
+    if (![_collectionConfigMap allKeys].count){
         [NSException raise: NSInternalInconsistencyException
                     format: @"%@", kCBLErrorMessageAccessDBWithoutCollection];
+    }
```

Comment:
This check can be removed entirely as collections will never be empty (it's asserted in `initWithCollections:target`.